deploy/alerts-preprod.yaml

```diff
@@ -26,11 +26,11 @@ spec:
       action: "Sjekk app {{ $labels.app }} i namespace {{ $labels.kubernetes_namespace }} sine selftest for å se hva som er galt"
     - alert: Høy andel HTTP serverfeil (5xx responser)
       severity: danger
-      expr: (100 * (sum by (backend) (rate(traefik_backend_requests_total{code=~"^5\\d\\d", backend=~"medlemskap-oppslag.nais.preprod.*"}[3m])) / sum by (backend) (rate(traefik_backend_requests_total{backend=~"medlemskap-oppslag.nais.preprod.*"}[3m])))) > 1
+      expr: (100 * (sum by (backend) (rate(traefik_backend_requests_total{code=~"^5\\d\\d", backend=~"https://medlemskap-oppslag.nais.preprod.local"}[3m])) / sum by (backend) (rate(traefik_backend_requests_total{backend=~"https://medlemskap-oppslag.nais.preprod.local"}[3m])))) > 1
```

Comment:
dobbeltsjekk med traefik_backend_request_total variabelen, men jeg tror ikke backend inneholder https://
tror det må fjernes. Legg også på en stjerne etter local i URL for å få med alle kall mot oss.